src/components/flow/routers/dial/helpers.ts

```diff
@@ -22,18 +22,24 @@ import { createUUID } from 'utils';
 export const nodeToState = (settings: NodeEditorSettings): DialRouterFormState => {
   let phone = '';
   let resultName: StringEntry = { value: '' };
+  let dialLimit = '';
+  let callLimit = '';
```

Comment:
these should have reasonable defaults

Reply:
@ericnewcomer i'm setting "defaults" although passively using the placeholder attr, but i can  set explicit defaults if that's preferred!